src/main/java/com/orasi/bluesource/Accounts.java

```diff
@@ -376,5 +396,137 @@ public void selectCell(int row, int column) {
 		PageLoaded.isDomComplete(driver, 1);
 	}
 	
+	/**
+	 * Clicks on New Project, fills out required fields and submits form
+	 * 
+	 * @author Christopher Batts
+	 */
+	public void addProject() {
+		btnNewProject.click();
+		txtProjectName.sendKeys("New Project 1");
+		txtProjectStart.sendKeys("04062018");
+		txtProjectEnd.sendKeys("06042018");
+		btnCreateProject.click();
+	}
```

Comment:
In general, unless it is a quick throwaway method, we don't hard code values.  You'll want to pass that in from the test method from a data provider or a data factory class.  This way the method can be data driven if needed.